apps/api/src/message-schemas/events/nsip-representation.schema.json

```diff
@@ -55,13 +55,13 @@
 			"type": "string",
 			"enum": ["PERSON", "ORGANISATION", "AGENT"]
 		},
-		"represented": {
```

Comment:
Could you please remove `represented` from the "required" array at the top of this file. And I think we need to add `representedId` and `representativeId` is to required?